packages/spans/src/index.ts

```diff
@@ -47,10 +57,10 @@ export const stopSpan = (
 export const addSpanEventToSpan = (
   spanId: string,
   name: string,
-  timeStampMs?: number,
```

Comment:
shouldn't this still be optional?

Reply:
Michiel asked to mark this mandatory fo both, since iOS won't work on it in 5.x